doc/advanced_configuration.rst

```diff
@@ -77,6 +77,39 @@ starting with ``plot_long_examples_``, you would do:
 As the patterns are parsed as `regular expressions`_, users are advised to consult the
 `regular expressions`_ module for more details.
 
+.. _sub_gallery_order:
+
+Sorting the subsections inside a gallery
+========================================
+
+Gallery subsections are sorted by default alphabetically by their folder
+name, and as such you can always organize them by changing your folder
+names. An alternative option is to use a sortkey to organize those
+subsections. We provide an explicit order sortkey where you have to define
+the order of all subfolders in your galleries.
+
+.. code-block:: python
+
+    from sphinx_gallery.sorting import ExplicitOrder
+    sphinx_gallery_conf = {
+        'examples_dirs': ['../examples','../tutorials'],
+        'subsection_order': ExplicitOrder(['../examples/sin_func',
+                                                 '../examples/no_output',
+                                                 '../tutorials/seaborn']),
+    }
+
+Here we build 2 main galleries `examples` and `tutorials`, each of them
+with subsections. To specify their order explicitly in the gallery we
+import :class:`sphinx_gallery.sorting.ExplicitOrder` and initialize
+it with the list of all subfolders with their paths relative to `conf.py`
+in the order you prefer them to appear. You can use Linux paths, and if you
+documentation is built in a Windows system, paths will be transformed to
+work accordingly, the converse does not hold.
```

Comment:
Yes. File-system absolute paths from windows will not work (and I guess in linux it won't work eithr). We require always absolute paths to the location of the conf.py